plugins/guests/debian/cap/configure_networks.rb

```diff
@@ -61,12 +61,21 @@ def self.configure_netplan(machine, interfaces, comm, networks)
           # By default, netplan expects the renderer to be systemd-networkd,
           # but if any device is managed by NetworkManager, then we use that renderer
           # ref: https://netplan.io/reference
-          renderer = NETPLAN_DEFAULT_RENDERER
```

Comment:
The `NETPLAN_DEFAULT_RENDERER` constant can be removed as well since this was the only place it was being used.